pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.depproj

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <!-- we intentionally don't want to produce output -->
+    <OutputPath>unused</OutputPath>
+    <!-- we don't want any analyzers by ResolveNuGetPackageAssets 
+         null-refs when this isn't set and an analyzer is in the packages -->
+    <Language>unused</Language>
+    <NuGetTargetMoniker>.NETCoreApp,Version=v1.1</NuGetTargetMoniker>
+    <PackageTargetFramework>netcoreapp1.1</PackageTargetFramework>
+    <PrimaryPackage>Microsoft.Private.CoreFx.NETCoreApp</PrimaryPackage>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="project.json" />
+  </ItemGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+
+  <Target Name="SetRuntimeProjectJson">
+    <!-- when building with a runtime, add it to the project.json file -->
+    <PropertyGroup>
+      <RuntimeIntermediateOutputPath>$(IntermediateOutputPath)$(NuGetRuntimeIdentifier)/</RuntimeIntermediateOutputPath>
+      <SourceProjectJson>$(ProjectJson)</SourceProjectJson>
+      <ProjectJson>$(RuntimeIntermediateOutputPath)project.json</ProjectJson>
+      <ProjectLockJson>$(RuntimeIntermediateOutputPath)project.lock.json</ProjectLockJson>
+    </PropertyGroup>
+  </Target>
+  
+  <Target Name="CreateRuntimeProjectJson"
+          Condition="'$(NuGetRuntimeIdentifier)' != ''"
+          DependsOnTargets="SetRuntimeProjectJson"
+          Inputs="$(SourceProjectJson)"
+          Outputs="$(ProjectJson)">
+    <PropertyGroup>
+      <_pjText>$([System.IO.File]::ReadAllText('$(SourceProjectJson)'))</_pjText>
+      <_ridInsertAt>$(_pjText.LastIndexOf('}'))</_ridInsertAt>
+      <_pjText>$(_pjText.Insert($(_ridInsertAt), '  ,"runtimes": { "$(NuGetRuntimeIdentifier)":{} }'))</_pjText>
```

Comment:
I changed this to use a similar technique, but I did have to change it to handle no RID.  PTAL